modules/sms/scripts/get_message.ts

```diff
@@ -0,0 +1,27 @@
+import { RuntimeError, ScriptContext } from "../module.gen.ts";
+import { Message } from "../utils/types.ts";
+import { TestProvider } from "../utils/providers/test.ts";
+import { TwilioProvider } from "../utils/providers/twilio.ts";
+import { SmsProvider } from "../utils/providers/index.ts";
+
+export interface Request {
+    id: string;
+}
+
+export type Response = Message;
+
+export async function run(
```

Comment:
this is too verbose. we need to provide a generic superset over other peoples' apis. it's better to expose _less_ than _more_ so we can maintain consistent behavior across providers. i'd recommend completely deleting this script unless you see a good reason otherwise.